### DISCUSSION TITLE
Reducing the logging when running elastalert in a container

### DISCUSSION THREAD

**Discussion by Shashankft9** [jertel/elastalert2#173] [Ideas]
Hey, I was wondering if anyone else has any opinions on reducing the number of logs that are generated by elastalert. 
So, I was running elastalert in a container, and I have made it to query the elasticsearch cluster every 1 minute and I only have 1 rule right now which isnt getting any hits and thats expected, but elastalert keeps on logging every minute atleast 6 log lines:
```
INFO:elastalert:Disabled rules are: []
INFO:elastalert:Sleeping for 59.999785 seconds
INFO:elastalert:Background configuration change check run at 2021-05-21 06:49 UTC
INFO:elastalert:Background alerts thread 0 pending alerts sent at 2021-05-21 06:49 UTC
INFO:elastalert:Queried rule helloworldrule_serverless from 2021-05-20 11:53 UTC to 2021-05-21 06:49 UTC: 62 / 62 hits
INFO:elastalert:Ran helloworldrule_serverless from 2021-05-20 11:53 UTC to 2021-05-21 06:49 UTC: 62 query hits (62 already seen), 0 matches, 0 alerts sent
INFO:elastalert:Disabled rules are: []
INFO:elastalert:Sleeping for 59.9996 seconds
INFO:elastalert:Background configuration change check run at 2021-05-21 06:50 UTC
INFO:elastalert:Background alerts thread 0 pending alerts sent at 2021-05-21 06:50 UTC
INFO:elastalert:Queried rule helloworldrule_serverless from 2021-05-20 11:53 UTC to 2021-05-21 06:51 UTC: 62 / 62 hits
INFO:elastalert:Ran helloworldrule_serverless from 2021-05-20 11:53 UTC to 2021-05-21 06:51 UTC: 62 query hits (62 already seen), 0 matches, 0 alerts sent
INFO:elastalert:Disabled rules are: []
INFO:elastalert:Sleeping for 59.999634 seconds
INFO:elastalert:Background configuration change check run at 2021-05-21 06:51 UTC
INFO:elastalert:Background alerts thread 0 pending alerts sent at 2021-05-21 06:51 UTC
INFO:elastalert:Queried rule helloworldrule_serverless from 2021-05-20 11:53 UTC to 2021-05-21 06:52 UTC: 62 / 62 hits
INFO:elastalert:Ran helloworldrule_serverless from 2021-05-20 11:53 UTC to 2021-05-21 06:52 UTC: 62 query hits (62 already seen), 0 matches, 0 alerts sent
```

Does anyone have any opinions on this, should all these be in some other logging level, lets say `debug`? And only the log lines of any error or success in post should be logged? And then through some config file this logging level can be modified? 

I believe this can be used to change logging level in elastalert: https://github.com/Yelp/elastalert/issues/2942#issuecomment-749077256 but this outputs the logs to a file? What If I dont want to output into a file, just change the logging level.

**Comment by jertel** [comment]
In your config file change all the `level: INFO` lines to `level:WARN`. Don't change anything else. Doing this will still output the logging to console but only if they are WARN or ERROR log lines.

Personally, as far as changing the 6 log lines to debug, I prefer to see that activity throughout the day. So I would not want to see everything changed to `debug`.

**Reply by Shashankft9** [reply]
thanks, makes sense. I will try the logging part.
